### DISCUSSION TITLE
Configure Your Node Express server with MVC Architecture using this package !

### DISCUSSION THREAD

**Discussion by NaNshekhar04** [nodejs/help#4701] [Show and tell]
Hey there, I wanted to share my new package with the community. The package helps you create node-express apps with MVC Folder Structure directly from your command-line. I am open to hearing your thoughts and working together to make this package better for everyone. Your feedback is valuable to me and I look forward to building a supportive community around this package.

https://www.npmjs.com/package/super-express-cli

**Comment by elementbound** [comment]
Hi @NaNshekhar04! 
This package looks pretty nice! Minor feedback: for me personally it would be nice to see at a quick glance ( for example on the package page ) what directory structure will be created.
It is pretty easy to find it out from the source but maybe this could help others who are interested in the package but don't feel like checking source / installing the package to find out.

**Reply by NaNshekhar04** [reply]
Heyy @elementbound 

Your feedback is appreciated & here is the updated package :

https://www.npmjs.com/package/super-express-cli

 You can check it out now & share what needs to be improved more : )
